packages/modul-components/src/components/option/option.html

```diff
@@ -1,33 +1,46 @@
-<m-popup class="m-option"
-         :class="{ 'm--is-disabled': disabled }"
-         :open.sync="open"
-         :padding="false"
-         :placement="placement"
-         :disabled="disabled"
-         :close-on-backdrop="true"
-         :background="false"
-         :shadow="false"
-         :focus-management="focusManagement"
-         @open="onOpen"
-         @close="onClose">
-    <m-icon-button :id="id"
-                   class="m-option__button"
-                   aria-haspopup="true"
-                   :aria-controls="ariaControls"
-                   :aria-expanded="open"
-                   slot="trigger"
-                   :skin="skin"
-                   :title="propTitle"
-                   icon-name="m-svg__options"
-                   :disabled="disabled"
-                   :button-size="size"
-                   @click="onClick">
-    </m-icon-button>
-    <ul :id="ariaControls"
-        class="m-option__list"
-        :style="{ maxHeight: menuMaxHeight }"
-        :aria-labelledby="id"
-        ref="menu">
-        <slot></slot>
-    </ul>
-</m-popup>
+<div class="m-option">
```

Comment:
Tel que discuté cette div là si tu peux l'enlever enlève là, sinon pas + grave que ça

Reply:
Comme discuté, je préfère quand chaque composant a son propre div, c'est plus clean.
D'autant plus qu'avant on avait : m-popup m-option m--is-inline sur le même div et c'est pas claire le modifier est pour qui...